webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtVmware.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { number_to_human_size as NumberToHumanSize } from 'number_helpers';
+import { translate as __ } from '../../../../../../common/I18n';
+
+const VirtVmware = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Datacenter')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.datacenter}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Cluster')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.cluster}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Memory')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {NumberToHumanSize(vm.memory_mb * 1024 ** 2, {
+          strip_insignificant_zeros: true,
+        })}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Public IP address')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.public_ip_address}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('MAC')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.interfaces_attributes[0].mac}
```

Comment:
I'm always worried about `null`. Can `vm.interfaces_attributes[0]` be `null` and crash the interface? Is there a safe navigator?

Reply:
Updated.